package.json

```diff
@@ -64,7 +65,8 @@
     "react-popper": "2.2.5",
     "react-tabs": "3.2.3",
     "react-transition-group": "4.4.2",
-    "relative-luminance": "2.0.1"
+    "relative-luminance": "2.0.1",
+    "wait-on": "6.0.1"
```

Comment:
these new ones are likely devDepenencies

Reply:
Still trying to figure out the CI here for this command:
`    - name: Serve Storybook and Run Tests 📖
        run: |
          npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
            "npx http-server storybook-static --port 6006 --silent" \
            "npx wait-on tcp:6006 && npm run test-storybook"`